client/app/scripts/superdesk-dictionaries/dictionaries.js

```diff
@@ -143,7 +151,7 @@
                 $scope.origDictionary = result;
                 $scope.dictionary = _.create(result);
                 $scope.dictionary.content = _.create(result.content || {});
-                $scope.dictionary.is_active = $scope.dictionary.is_active !== false;
+                $scope.dictionary.is_active = $scope.dictionary.is_active !== 'false';
```

Comment:
Hm... I think this should be handled on the server then, if it is not too much work. Having semantically "incorrect" data types makes it harder to work with such objects elsewhere...